terraform.tfvars

```diff
@@ -1,46 +1,45 @@
 landing_zone_providers = {
   default = {
-    account_id = "123456789012"
-    region     = "us-east-1"
+    account_id = "914196360175"
+    region = "us-east-1"
   }
 
-  # ,
-  # sample = {
-  #   account_id = "123456789012"
-  #   region     = "us-east-1"
-  # }
+//  sample = {
```

Comment:
revert back `#` and change account_id to `987654321098`

Reply:
updated